src/main/java/uk/gov/hmcts/reform/bulkscan/helper/BulkScanTransformHelper.java

```diff
@@ -0,0 +1,42 @@
+package uk.gov.hmcts.reform.bulkscan.helper;
+
+import lombok.extern.slf4j.Slf4j;
+import org.apache.commons.lang3.StringUtils;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+@Slf4j
+@SuppressWarnings("PMD.AvoidReassigningParameters")
+public final class BulkScanTransformHelper {
+
+    private BulkScanTransformHelper() {
+
+    }
+
+    @SuppressWarnings("unchecked")
+    public static Object getMapObjectAndValue(Object object, Map<String, String> inputFieldsMap) {
+        List<Object> list = new ArrayList<>();
+        if (object instanceof Map) {
+            Map<String, Object> innerMap = (Map<String, Object>) object;
```

Comment:
if possible variable name can be changed to more meaningful names

Reply:
it changed.